unity-renderer/Assets/UnityGLTF/Scripts/Loader/WebRequestLoader.cs

```diff
@@ -84,33 +84,32 @@ private async UniTask CreateHTTPRequest(string rootUri, string httpRequestPath,
                 requestAttemps: 3);
 
             Assert.IsNotNull(asyncOp, "asyncOp == null ... Maybe you are using a mocked WebRequestController?");
-            
+
             token.ThrowIfCancellationRequested();
-            
+
             await UniTask.WaitUntil( () => asyncOp.isDone || asyncOp.isDisposed || asyncOp.isSucceeded, cancellationToken: token);
 
 #if UNITY_STANDALONE || UNITY_EDITOR
             if (DataStore.i.common.isApplicationQuitting.Get())
                 return;
 #endif
-            
+
             token.ThrowIfCancellationRequested();
-            
+
             bool error = false;
             string errorMessage = null;
 
             if (!asyncOp.isSucceeded)
             {
-                Debug.Log($"{asyncOp.webRequest.error} - {finalUrl} - responseCode: {asyncOp.webRequest.responseCode}");
-                errorMessage = $"{asyncOp.webRequest.error} {asyncOp.webRequest.downloadHandler.text}";
-                error = true;
+                errorMessage = $"{asyncOp.webRequest.error} {asyncOp.webRequest.downloadHandler.text} {finalUrl}";
+                throw new Exception(errorMessage);
```

Comment:
Does it make sense to introduce a separate exception just like you did in other classes?